<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WydatkiDialog</class>
 <widget class="QDialog" name="WydatkiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WYDATKI</string>
  </property>
  <widget class="QPushButton" name="wsteczWY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DODAJ NOWY WYDATEK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kategorieWY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Transport</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inwestowanie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oszczędności</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mieszkanie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Żywność</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rozrywka</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Osobiste</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Finanse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inne</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>KATEGORIA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>KWOTA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kwotaWY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zł</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dataWY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OPIS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="opisWY">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="caloscWY">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TWOJE PIENIADZE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zł</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dodajWY">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>DODAJ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wsteczWY</sender>
   <signal>clicked()</signal>
   <receiver>WydatkiDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
